src/plots/base.tsx

```diff
@@ -50,15 +65,25 @@ export default class BaseChart<
   componentDidUpdate() {
     const config = this.getConfig(this.props)
     const { data, ...restConfig } = config as any
-    const isConfigChanged = !isEqual(this.config, restConfig)
+    console.log(data)
+    console.log(this.config.data)
+    const thisFnKeys = this.getFnKeys(this.config)
+    const restFnKeys = this.getFnKeys(restConfig)
+    const isConfigChanged = !isEqual(
+      omit(this.config, thisFnKeys),
+      omit(restConfig, restFnKeys)
+    )
+    const dataChanged = !isEqual(data, this.data)
```

Comment:
I don't think it is necessary to compare `data`

when the amount of data is large, this comparison can take a lot of time

Reply:
Ok sure.